lib/shared/src/editor/displayPath.ts

```diff
@@ -150,7 +150,7 @@ function _displayPath(
  * tests.
  */
 function fixPathSep(fsPath: string, isWindows: boolean, scheme: string): string {
-    return isWindows && scheme === 'file' ? fsPath.replaceAll('/', '\\') : fsPath
+    return isWindows && scheme === 'file' ? fsPath.replaceAll('/', '\\').replace(/^\\/, '') : fsPath
```

Comment:
```suggestion
    return isWindows && scheme === 'file' ? fsPath.replaceAll('/', '\\').replace(/^\\/, '') : fsPath
```

This removes leading dashes from file path on Windows for display path only.

Reply:
This doesn't look right.

Windows has UNC paths that start with `\\`, like `\\.\c:\fooba~12.txt`, `\\hostname\foo\bar.txt` etc. We should not be deleting important parts of paths.